[nextion] Add configurable limit for commands processed per loop (#4963)
OttoWinter/esphomedocs: components/display/nextion.rst

- **command_spacing** (*Optional*, :ref:`config-time`): Sets the minimum time between commands sent to the Nextion display.
  A higher value can help prevent buffer overflows but will result in slower interface updates.
  Range is ``0-255ms``. Defaults to ``0ms`` (disabled).
- **max_commands_per_loop** (*Optional*, integer): Limits the number of commands processed per loop cycle.  
  This helps prevent stack overflows when a large number of commands are queued. 

>   Lower values (e.g., ``20``) can help improve system stability in constrained environments.

Reviewer: ```suggestion
  Lower values (for example, ``20``) may help improve stability in constrained environments.
```
